pytest_tests/testsuites/services/http_gate/test_http_object.py

```diff
@@ -0,0 +1,169 @@
+import logging
+import os
+from random import choice
+from time import sleep
+
+import allure
+import pytest
+from common import COMPLEX_OBJ_SIZE, HTTP_GATE
+from container import create_container
+from epoch import get_epoch, tick_epoch
+from file_helper import generate_file, get_file_hash
+from neofs_testlib.shell import Shell
+from python_keywords.http_gate import (
+    get_via_http_curl,
+    get_via_http_gate,
+    get_via_http_gate_by_attribute,
+    get_via_zip_http_gate,
+    upload_via_http_gate,
+    upload_via_http_gate_curl,
+)
+from python_keywords.neofs_verbs import get_object, put_object
+from python_keywords.storage_policy import get_nodes_without_object
+from utility import wait_for_gc_pass_on_storage_nodes
+from wellknown_acl import PUBLIC_ACL
+
+logger = logging.getLogger("NeoLogger")
+OBJECT_NOT_FOUND_ERROR = "not found"
```

Comment:
at pytest_tests/helpers/grpc_responses.py we have OBJECT_NOT_FOUND, is this the same output?